pallets/utxo/src/lib.rs

```diff
@@ -386,7 +386,7 @@ pub mod pallet {
     #[pallet::storage]
     #[pallet::getter(fn utxo_store)]
     pub(super) type UtxoStore<T: Config> =
-        StorageMap<_, Identity, H256, Option<TransactionOutputFor<T>>, ValueQuery>;
+        StorageMap<_, Identity, H256, TransactionOutputFor<T>, OptionQuery>;
```

Comment:
Hmm now I see `OptionQuery` is the default for `StorageMap` so can get rid of the type param completely

Reply:
i see. well, i'll change what's mine, first,